deep-learning/_posts/2017-10-18-SR-weakly-supervised-joint-conv-sparse-coding.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: review
+title: "Simultaneous Super-Resolution and Cross-Modality Synthesis of 3D Medical Images using Weakly-Supervised Joint Convolutional Sparse Coding"
+tags: deep-learning MRI
+author: "Philippe Poulin"
+cite:
+    authors: "Y. Huang, L. Shao, A. F. Frangi"
+    title:   "Simultaneous Super-Resolution and Cross-Modality Synthesis of 3D Medical Images using Weakly-Supervised Joint Convolutional Sparse Coding"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1705.02596.pdf"
+---
+
+## Summary
+
+Acquisition of high-resolution, multimodal MRI images is long and costly. The authors propose a model to do both super-resolution and cross-modality image synthesis from low resolution images. Their method is called WEENIE (WEakly-supErvised joiNt convolutIonal sparsE coding).
+
+More precisely, the goal is to be able to reconstruct a high resolution image from a low resolution image of different modality using Convolutional Sparse Coding (CSC).
+
+![](/deep-learning/images/sr-weak-conv-sparse/figure1.png)
+
+The objective of CSC is to learn $$K$$ filters $$\mathbf f_k$$ and associated sparse feature maps $$\mathbf z_k$$ such that the input $$\mathbf x$$ can be reconstructed from the sum of the convolution of each pair of filter and sparse feature map. A constraint is added on the filters to make sure one filter does not absorb all the energy of the system.
```

Comment:
and sparse feature map**s** ?

Reply:
Each pair contains a single filter and a single "feature map" (They call it feature map, but it is more like a dictionary entry, which is used by a specific filter to reconstruct an image)